components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json

```diff
@@ -20,15 +22,10 @@
         "xdm:personalFinances": {
           "type": "object",
           "properties": {
-            "xdm:creditLimitIncreaseDetails": {
```

Comment:
We still need nesting here right ? "personalFinances" ==> "creditLimitIncreaseDetails" ?

Reply:
Yes, this level of nesting would keep it consistent.